ci/src/artifacts/upload.sh

```diff
@@ -25,31 +25,39 @@ if [ "$(basename $f)" == "SHA256SUMS" ]; then
     cat "$f" >&2
 fi
 
+# Multipart upload does not work trough Cloudflare for some reason.
+# Just disabling it with `--s3-upload-cutoff` for now.
```

Comment:
Nitpicking: this comment only seems to apply to the `--s3-upload-cutoff=5G` flag. So you could consider moving it on top of that flag.